src/act.other.cpp

```diff
@@ -4165,6 +4165,128 @@ ACMD(do_memory)
   }
 }
 
+// Looks for any working cyberdeck in the character's inventory or worn equipment.
+struct obj_data *find_cyberdeck(struct char_data *ch)
+{
+  struct obj_data *deck;
```

Comment:
Nit: Unless you're going to use this outside of its for-loop scope, this is best defined as part of the for-loop instantiation (e.g. `for (struct obj_data *deck = ch->carrying...`). Keeping vars to the scope where they're used makes maintaining the code easier in the long term.

Reply:
👍 Ah yeah in this case it's referenced in both for loops, but actually it doesn't need to be